README.md

```diff
@@ -37,13 +37,13 @@ gem install active_outbox
 
 ## Usage
 ### Setup
-Create an `Outbox` table using the provided generator and corresponding model. Any model name can be passed as an argument but if empty it will default to just `Outobx`. The generated table name will be `model_name_outboxes`.
+Create an initializer under `config/initializers/active_outbox.rb` and setup the default outbox class to the new `Outbox` model you just created.
```

Comment:
This is not clear to me. The `Outbox` model was already created? don't you need to run the model generator for that? Maybe the order of these steps is wrong

Reply:
good catch, that was left after modifying the initial message